examples/teaching-vm_openrefine/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Integrated example for the Ten Simple Rules for Dockerfiles project, 
+# see https://osf.io/fsd7t/
+# This Dockerfile is a rewrite based on
+# - https://github.com/innovationOUtside/tm351vm/blob/master/build/minimal/Dockerfile
+# - https://github.com/innovationOUtside/tm351vm/blob/master/build/openrefine/Dockerfile
+FROM ubuntu:16.04
```

Comment:
Wouldn’t it be bad to show a base for an OS that is reaching end of life soon?

Reply:
I did only improvements based on the rules, I think changes in content are out of scope for the before-after - examples. Maybe we should clarify that in the base README?

Reply:
I think someone (in the future) might come and say "why is there an example for an OS that isn't supported anymore) but in that it's not completely at end of life, and it is out of scope, I'm also fine leaving this for now.

Reply:
You're right that we must expect readers to use these files as templates, so I added a note to the examples' README and extended the documentation a bit.